aurora-spring-boot-webflux-starter/src/main/java/no/skatteetaten/aurora/webflux/config/WebFluxStarterApplicationConfig.java

```diff
@@ -39,4 +45,18 @@ public HttpRequestParser sleuthHttpServerRequestParser() {
     public AuroraSpanHandler auroraSpanHandler() {
         return new AuroraSpanHandler();
     }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "aurora.webflux.trace.auth", name = { "username", "password" })
+    public ZipkinWebClientBuilderProvider zipkinWebClientBuilderProvider(
+        @Value("aurora.webflux.trace.auth.username") String username,
+        @Value("aurora.webflux.trace.auth.password") String password,
+        WebClient.Builder builder
+    ) {
+        return () -> {
+            String encodedAuth =
+                Base64Utils.encodeToString((username + ":" + password).getBytes(StandardCharsets.UTF_8));
+            return builder.defaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + encodedAuth);
```

Comment:
Kan man bruke denne metoden? https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpHeaders.html#setBasicAuth-java.lang.String-java.lang.String-java.nio.charset.Charset-

Reply:
Visste jeg ikke om, takk for tipset!